src/Moryx.CommandCenter.Web/src/databases/container/DatabaseModel.tsx

```diff
@@ -82,18 +78,24 @@ class DatabaseModel extends React.Component<DatabaseModelPropsModel & DatabaseMo
         this.onTestConnection();
     }
 
+    public componentDidUpdate(prevProps: DatabaseModelPropsModel, prevState: DatabaseModelStateModel): void {
```

Comment:
The `prevState` parameter is not used is it?

Reply:
No, it is not, but the interface requires it.